### PR TITLE
MOB-17719/Bundled-Rules-section-doesn't-match-with-old-docs

### PR DESCRIPTION
**Pull request by hbhandal80**
## Description

Updated to include android zip file placement location.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/mobile-core/rules-engine/index.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ On older versions of Experience Platform Mobile SDKs (prior to iOS version 1.6.2
```
Comment (praveek) [inline src/pages/documentation/mobile-core/rules-engine/index.md:55]: @cdhoffmann This is supported both in iOS and Android. We should remove `(iOS AEP 3.x Only)`

Comment (hbhandal80) [reply src/pages/documentation/mobile-core/rules-engine/index.md:55]: (iOS AEP 3.x Only) has been removed.

Comment (emdobrin) [reply src/pages/documentation/mobile-core/rules-engine/index.md:55]: @hbhandal80 can you also include an info block under this title with the text 
```
This feature is available from iOS AEP 3.x and Android 1.11.2
```

File: src/pages/documentation/mobile-core/rules-engine/index.md [hunk 2]
```diff
@@ -51,11 +51,13 @@ Conditions can be one match or a group of matches.
...
 
-## Using Bundled Rules (iOS AEP 3.x Only)
+## Using Bundled Rules
 
 In addition to the remote configuration, you can also include a rules zip file in your app bundle to be used by the SDK before rules have been downloaded from the Data Collection UI. To add bundled rules to your app, follow these steps:
 1. Download the rules zip file using the following URL: `https://assets.adobedtm.com/PASTE-ENVIRONMENT-ID-rules.zip` replacing `PASTE-ENVIRONMENT-ID` with your mobile property environment ID. 
-2. Rename the zip file to "ADBMobileConfig-rules.zip" and place the zip anywhere that it is accessible in your app bundle.
+2. Rename the zip file to "ADBMobileConfig-rules.zip" and place the it in the relevant location for your device:
+- IOS: Place the zip anywhere that it is accessible in your app bundle.
```
Comment (emdobrin) [inline src/pages/documentation/mobile-core/rules-engine/index.md:59]: typo: iOS

Comment (hbhandal80) [reply src/pages/documentation/mobile-core/rules-engine/index.md]: Updated to iOS

File: src/pages/documentation/mobile-core/rules-engine/index.md (hunk 1 above)
Comment (hbhandal80) [reply src/pages/documentation/mobile-core/rules-engine/index.md:59]: Note added.

File: src/pages/documentation/mobile-core/rules-engine/index.md [hunk 3]
```diff
@@ -51,11 +51,17 @@ Conditions can be one match or a group of matches.
 
 On older versions of Experience Platform Mobile SDKs (prior to iOS version 1.6.2, Android version 1.5.4), after the first launch of the app, it always takes some time to download the rules from the remote servers. During this time, Rules Engine won't be able to evaluate the first several events until the rules are loaded. Starting from iOS version 1.6.2 and Android version 1.5.4, we add the capability to cache the events before the rules are downloaded and will re-evaluate those events afterward. This change is mainly to enable the trigger of Postback based on the install event.
 
-## Using Bundled Rules (iOS AEP 3.x Only)
+## Using Bundled Rules
+
+>[!NOTE]
+>
+>This feature is available from iOS AEP 3.x and Android 1.11.2.
```
Comment (emdobrin) [inline src/pages/documentation/mobile-core/rules-engine/index.md:58]: We should use the gatsby format for info notes here:
```
<InlineAlert variant="info" slots="text"/>
This feature is available from iOS AEP 3.x and Android 1.11.2.
```

Comment (hbhandal80) [reply src/pages/documentation/mobile-core/rules-engine/index.md]: Replace with an InlineAlert.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**